domains/spideybot.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Vishvaas06",
+           "email": "vishvaaskumar06@gmail.com",
+           "discord": "523154428576923679"
+        },
+    
+        "record": {
+            "TXT": "5cc962eac4d053f1a353cc209a6b84"
```

Comment:
TXT record alone doesn't do anything. Can you provide some clarification on why this file only contains a TXT record?